### ISSUE TITLE
Plasma Cutter Is a Piece of Shit ayo

### ISSUE THREAD

**Issue by Tk420634** [Tk420634/Para-Arf#55]
**Problem Description**:
The plasma cutter is shit.

**What did you expect to happen**: 
For it to fire and do more than 1 tile of mining.

**What happened instead**:
It did one fucking tile. Reeeeee.

**Why is this bad/What are the consequences:**
3 > 1, 5 > 1

**Steps to reproduce the problem**:
Be Miner
Get Plasma Cutter
Be Disappointed

**Comment by jonathanhybrid** [comment]
I dont have perms to tag this as SOLVED.

**Comment by TalkingCactus** [comment]
Did you test it?

**Comment by jonathanhybrid** [comment]
Yeah it didn't work, strange... I need to replace the on_hit function completely with a new one, or change how this projectile works.

**Comment by TalkingCactus** [comment]
I already swapped it for TG's version, but it didn't work as expected.

**Comment by jonathanhybrid** [comment]
It's the . = ..() causing the deletion. btw.

**Comment by TalkingCactus** [comment]
![image](https://user-images.githubusercontent.com/8886619/33904423-7c46012a-df49-11e7-8104-2afa403b0be9.png)
I know, but I didn't actually try to fix it after this point.
